recipes/modvis/meta.yaml

```
@@ -0,0 +1,58 @@
+# Everytime you make a release, please update the version number below and the sha256 hash
+{% set name = "modvis" %}
+{% set version = "0.0.9" %} 
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  # git_rev: v0.0.9
+  # git_url: https://github.com/pinshuai/modvis.git
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+
+  # the sha256 hash can be obtained by running `sha256sum <modvis-xxx.tar.gz>`
+  sha256: 4c9e27dd7409ce2b23eb6b3e5d1385a79f3326bacb78650304f4dfaf6b51e53c
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv "
+
+requirements:
+  host:
+    - python
+    - pip
+    - setuptools
+    - wheel
+  run:
+    - python
+    - numpy
+    - matplotlib-base
+    - scipy
+    - pandas
+    - h5py
+    - seaborn
+    - scikit-learn
+    - dataretrieval
+    - geopandas
+
+
+test:
+  imports:
+    - modvis
+    - modvis.ats_xdmf
+    - modvis.ATSutils
+    - modvis.general_plots
+    - modvis.plot_vis_file
+    - modvis.utils
+
+about:
+  home: https://pinshuai.github.io/modvis
```

```suggestion
  home: https://pinshuai.github.io/modvis
  dev_url: https://github.com/pinshuai/modvis
```
This adds the `modvis` GitHub URL from before